make/autoconf/jdk-options.m4

```diff
@@ -684,7 +684,7 @@ AC_DEFUN([JDKOPT_ENABLE_DISABLE_CDS_ARCHIVE],
 #
 AC_DEFUN([JDKOPT_ENABLE_DISABLE_COMPATIBLE_CDS_ALIGNMENT],
 [
-  UTIL_ARG_ENABLE(NAME: compatible-cds-alignment, DEFAULT: $COMPATIBLE_CDS_ALIGNMENT_DEFAULT,
+  UTIL_ARG_ENABLE(NAME: compatible-cds-alignment, DEFAULT: false,
```

Comment:
I suggest a better solution would be to ensure that `COMPATIBLE_CDS_ALIGNMENT_DEFAULT=false` so `AC_SUBST(COMPATIBLE_CDS_ALIGNMENT_DEFAULT)` is consistent should `COMPATIBLE_CDS_ALIGNMENT_DEFAULT` be used elsewhere. See lines 202-206.

Reply:
Updated.